src/Libraries/RevitNodes/Application/Document.cs

```diff
@@ -84,19 +86,25 @@ public DynamoUnits.Location Location
         /// <summary>
         /// Saves all of the input families at a given location.
         /// </summary>
-        /// <param name="family">The families to save</param>
-        /// <param name="directoryPath">Directory to save the families in</param>
+        /// <param name="family">The Revit family to save</param>
+        /// <param name="directoryPath">Directory to save the family to. If directory does not exist, it will be created.</param>
         /// <returns>File path of saved families</returns>
-        public string SaveFamilyLibrary(Family family, string directoryPath)
+        public string SaveFamilyLibraryToFolder(Family family, string directoryPath)
         {
+            var dir = new DirectoryInfo(directoryPath);
```

Comment:
what happens when this fails (ie we give it a bad path) ? if the error makes sense, it's fine to leave as is, but if it makes it hard for users to understand that's the issue, let's wrap this in a try catch and throw a more intelligible error.

Reply:
been testing this but not sure what a bad path could be. The only thing I can find is if you try to create a directory with illegal characters (`*` `.` `"` `/` `\` `[` `]` `:` `;` `|` `,`)

If that happens the exception thrown is this, which I think is an easy to understand error?
![image](https://user-images.githubusercontent.com/13732445/71518361-673fb080-28aa-11ea-84ec-1cdd383ae4dd.png)

Reply:
Error message on a bad path
![image](https://user-images.githubusercontent.com/13732445/71715606-de74c780-2e09-11ea-9f91-bb66eaff5358.png)